projects/experimental/components/badge-alert/badge-alert.style.less

```diff
@@ -0,0 +1,58 @@
+@import 'taiga-ui-local';
+
+tui-badge-alert {
+    position: relative;
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    color: var(--tui-base-01);
+    border-radius: 2rem;
+    white-space: nowrap;
+    overflow: hidden;
+    vertical-align: middle;
+    font: var(--tui-font-text-s);
+    max-width: 100%;
+    padding: 0 0.25rem;
+    background: #f52222;
+
+    &[data-size='l'] {
+        height: 1.5rem;
```

Comment:
We can add `--t-size` variable and change it, so that it's less lines of CSS as a result